### PR TITLE
ui: update bottom tab animation & handle navigate after adding widget

### PR DESCRIPTION
**Pull request by dev-tronghieu**

### CODE REVIEW

File: packages/engine/state/widget/shared.ts
Comment (dev-tronghieu) [inline packages/engine/state/widget/shared.ts:1]: forgive me pls, i know this commit should not be included in this #PR

File: packages/engine/state/widget/shared.ts [hunk 1]
```diff
@@ -109,10 +109,10 @@ export const mockWidgets: WidgetDocument[] = [
 		widgetType: 'Layout',
 		timestamp: new Date().toISOString(),
 		storeMeta: {
-			iconUri: '/img/explore/logo-aptos.svg',
+			iconUri: '/img/explore/logo-aptos.png',
 			iconSize: 26,
 			iconColor: 'black',
-			coverUri: '/img/explore/thumbnail-aptos.svg',
+			coverUri: '/img/explore/thumbnail-aptos.png',
```
Comment (dev-tronghieu) [inline packages/engine/state/widget/shared.ts:115]: forgive me pls, i know this commit should not be included in this #pr

File: apps/mobile/src/screens/Dashboard/Home/Widget.tsx [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
-import ExplorerScreen from '../Explore';
-
 export const WidgetScreen = () => {
+	const insets = useSafeAreaInsets();
 	const { activeWidgetId } = useSnapshot(appState);
+	const explorerContainerStyle: ViewStyle = {
+		paddingTop: insets.top,
+	};
+	const explorerScrollStyle: ViewStyle = {
+		paddingBottom: insets.bottom + insets.bottom,
```
Comment (dev-tronghieu) [inline apps/mobile/src/screens/Dashboard/Home/Widget.tsx:17]: why there is a `insets.bottom + insets.bottom`?

Comment (fluctlight-kayaba) [reply apps/mobile/src/screens/Dashboard/Home/Widget.tsx:17]: oops, nice catch lol

**Review by fluctlight-kayaba — APPROVED** [review]

**Pull request merged**